packages/net-stubbing/lib/server/util.ts

```diff
@@ -18,12 +19,43 @@ import type { CypressIncomingRequest } from '@packages/proxy'
 import type { InterceptedRequest } from './intercepted-request'
 import { caseInsensitiveGet, caseInsensitiveHas } from '../util'
 
-// TODO: move this into net-stubbing once cy.route is removed
```

Comment:
There was a note about moving this into net-stubbing once cy.route was removed, so I did that 🤷🏻‍♀️